### PR TITLE
Use class namespace instead of string in testing.

### PR DESCRIPTION
**Pull request by khoinv**

### CODE REVIEW

**Comment by guillaumebriday** [comment]
What’s the benefit?

**Comment by khoinv** [comment]
I want to organize my project into two contexts likes account (all user-related logic) and blog context. When I do that, I switch the Post from Models/Post to Models/Blog/Post and I want to all testcases will pass without any fixing in these testcases.

File: tests/Feature/Api/V1/CommentTest.php [hunk 1]
```diff
@@ -139,11 +139,12 @@ public function testStore()
 
     public function testStoreFail()
     {
+        $model = Post::class;
         $this->actingAsUser('api')
             ->json('POST', "/api/v1/posts/31415/comments", $this->validParams())
             ->assertNotFound()
             ->assertJson([
-                'message' => 'No query results for model [App\\Models\\Post] 31415'
+                'message' => "No query results for model [{$model}] 31415"
```
Comment (guillaumebriday) [inline tests/Feature/Api/V1/CommentTest.php:147]: I think the temp variable is useless with this: `{Post::class}`

**Comment by guillaumebriday** [comment]
Ok why not 👍

File: tests/Feature/Api/V1/CommentTest.php (hunk 1 above)
Comment (khoinv) [reply tests/Feature/Api/V1/CommentTest.php:147]: I also tried it before. But unfortunately, in all php versions, string interpolation does not support that syntax.

Comment (guillaumebriday) [reply tests/Feature/Api/V1/CommentTest.php:147]: Ah ok.

I'm not a huge fan of temp variables.

I find this more elegant:
```php
'message' => sprintf('No query results for model [%s] 31415', Post::class)
``

Comment (guillaumebriday) [reply tests/Feature/Api/V1/CommentTest.php:147]: ping @khoinv

Comment (khoinv) [reply tests/Feature/Api/V1/CommentTest.php]: I'm sorry for the late reply. I agree with you about the ugly of using temporary variables.
I fixed and pushed.

**Pull request merged**